core/alsp_src/builtins/sio.pro

```diff
@@ -2788,6 +2782,7 @@ push_prompt(Stream_or_alias)
 	:-
 	is_stream(Stream_or_alias,Stream),
 	stream_pgoals(Stream,PromptGoal),
+push_prompt('push_prompt: PromptGoal'=PromptGoal),pbi_nl,
```

Comment:
Yes, though totally confused.  Clearly never ran.

> On Sep 9, 2016, at 5:45 PM, Chuck Houpt <notifications@github.com> wrote:
> 
> In core/alsp_src/builtins/sio.pro:
> 
> > @@ -2788,6 +2782,7 @@ push_prompt(Stream_or_alias)
> >  	:-
> >  	is_stream(Stream_or_alias,Stream),
> >  	stream_pgoals(Stream,PromptGoal),
> > +push_prompt('push_prompt: PromptGoal'=PromptGoal),pbi_nl,
> 
> Is this debug code?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>